pages/junction/canvas_course_add_user_page.rb

```diff
@@ -11,7 +11,7 @@ class CanvasCourseAddUserPage
       include Page
       include JunctionPages
 
-      button(:maintenance_button, class: 'bc-template-canvas-maintenance-notice-button')
+      button(:maintenance_button, xpath: '//button[contains(@class, "bc-template-canvas-maintenance-notice-button")]')
```

Comment:
PR https://github.com/ets-berkeley-edu/calcentral/pull/7953 puts an ID `maintenance-notice-button` on this button